livox_ros_driver/CMakeLists.txt

```diff
@@ -104,6 +111,11 @@ if((NOT LIVOX_SDK_LIBRARY) OR (NOT EXISTS ${LIVOX_SDK_LIBRARY}))
 		MESSAGE(${res})
 	ENDFOREACH()
 
+	# add position independent flag to cmakelist of sdk_core CMakeLists.txt if not already added.
+	execute_process(COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/set_position_independent_flag.sh
```

Comment:
why don't we just update the Livox SDK repo to add this flag?

Reply:
The problem is that this change happens in the Livox SDK code.

Reply:
We've already forked the Livox-SDK library. So I don't see why we couldn't just update it there? That would simplify this code a bit.

https://github.com/oxin-ros/Livox-SDK

Reply:
I've gone ahead and updated the Livox SDK to make it a catkin package: 

https://github.com/oxin-ros/Livox-SDK/pull/1

we can just use that.